### DISCUSSION TITLE
Zigbee model 'undefined' and manufacturer name 'undefined'"

### DISCUSSION THREAD

**Discussion by eliasjonsson023** [Koenkk/zigbee2mqtt#7099] [Devices]
I have an Ikea LED1732G11 that has worked fine for some time now. Then I decided to upgrade the firmware of the CC2531 from 2018 version to the latest 2020 version. The update was successful. Then I had to re-pair all the things, among them was an Ikea motion detector. Finally I was able to see the Occupancy: True reports from the Ikea motion detector, something that didn't show up on the 2018 firmware version. So far so good.
I paired the Ikea LED1732G11 again. From what I can remember all went well, except from when it was time to do something with it. The log output said something about ModelID: undefined and I couldn't even toggle the on/off state of the light. 
Nothing about the LED1732G11 works anymore.

This is what happens when I try to update the firmware of the LED1732G11 light source:

`{"commit":"b6085fc","coordinator":{"meta":{"maintrel":3,"majorrel":2,"minorrel":6,"product":0,"revision":20201127,"transportrev":2},"type":"zStack12"},"log_level":"info","network":{"channel":11,"extendedPanID":"0xdddddddddddddddd","panID":6754},"permit_join":false,"version":"1.18.1"}
[]
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{\"data\":{\"friendly_name\":\"klaedkammare\",\"ieee_address\":\"0x5c0272fffe9d8095\"},\"type\":\"device_announce\"}'"}
{"data":{"friendly_name":"klaedkammare","ieee_address":"0x5c0272fffe9d8095"},"type":"device_announce"}
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{\"message\":\"announce\",\"meta\":{\"friendly_name\":\"klaedkammare\"},\"type\":\"device_announced\"}'"}
{"message":"announce","meta":{"friendly_name":"klaedkammare"},"type":"device_announced"}
{"level":"warning","message":"Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'"}
{"level":"warning","message":"Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html."}
{"level":"warning","message":"Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'"}
{"level":"warning","message":"Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html."}
{"id":"klaedkammare"}
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{\"message\":\"Device 'klaedkammare' does not support OTA updates\",\"meta\":{\"device\":\"klaedkammare\",\"status\":\"not_supported\"},\"type\":\"ota_update\"}'"}
{"message":"Device 'klaedkammare' does not support OTA updates","meta":{"device":"klaedkammare","status":"not_supported"},"type":"ota_update"}
{"level":"info","message":"MQTT publish: topic 'zigbee2mqtt/bridge/response/device/ota_update/check', payload '{\"data\":{\"id\":\"klaedkammare\"},\"error\":\"Device 'klaedkammare' does not support OTA updates\",\"status\":\"error\"}'"}
{"data":{"id":"klaedkammare"},"error":"Device 'klaedkammare' does not support OTA updates","status":"error"}
{"level":"error","message":"Device 'klaedkammare' does not support OTA updates"}
{"level":"warning","message":"Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'"}
{"level":"warning","message":"Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html."}
{"level":"warning","message":"Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'"}
{"level":"warning","message":"Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html."}
{"level":"warning","message":"Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'"}
{"level":"warning","message":"Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html."}
`

and this is when I tell the light ot switch off:
`{"state":"off"}
{"level":"warning","message":"Device with modelID 'undefined' is not supported."}
{"level":"warning","message":"Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html"}
`

Any ideas on how to fix this issue?

**Comment by Twit123** [comment]
It's paired badly, remove the device (force remove) and pair it again.

Simon

**Reply by eliasjonsson023** [reply]
I removed the device and then paired it again. Now it is possible to control the light source again. Thank you!
